src/sentry/tasks/reports.py

```diff
@@ -582,7 +584,8 @@ def prepare_reports(dry_run=False, *args, **kwargs):
                     "organization_id": organization.id,
                 },
             )
-        prepare_organization_report.delay(timestamp, duration, organization.id, dry_run=dry_run)
+
+            prepare_organization_report.delay(timestamp, duration, organization.id, dry_run=dry_run)
```

Comment:
Won't this turn off weekly reports for any org that isn't flagged in to report-debugging?

Reply:
Yea, that's the intention. So that we don't keep sending emails every 30 minutes to real customers. My thinking is we'll revert this once things are fixed.